### PR TITLE
Remove unwanted or unused Rpsec tests

### PR DESCRIPTION
**Pull request by jasonBirchall**
Overview
---

This PR removes a number of rspec tests from our suite of tests. The tests removed here have either been made redundant by the move to EKS OR they are covered elsewhere.

### CODE REVIEW

File: smoke-tests/spec/alertmanager_spec.rb [hunk 1]
```diff
@@ -1,12 +0,0 @@
-require "spec_helper"
```
Comment (poornima-krishnasamy) [inline smoke-tests/spec/alertmanager_spec.rb:1]: Do we have an identical alertmanager check in go?

Comment (jasonBirchall) [reply smoke-tests/spec/alertmanager_spec.rb:1]: This particular check confirms if the correct servicemonitor exists: We do it slightly differently in Go, checking if services and servicemonitors exist in bulk
https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/main/test/config/config.go#L102-L111

File: smoke-tests/spec/prometheusrules_spec.rb [hunk 2]
```diff
@@ -1,39 +0,0 @@
-require "spec_helper"
```
Comment (poornima-krishnasamy) [inline smoke-tests/spec/prometheusrules_spec.rb:1]: And this one as well as above? Even if there is a serviceMonitor, should we not check if the rules are present?

File: smoke-tests/spec/certificates_spec.rb [hunk 3]
```diff
@@ -1,12 +0,0 @@
-require "spec_helper"
```
[the remark this answers is not in the document]
Comment (jasonBirchall) [reply smoke-tests/spec/certificates_spec.rb:1]: New test added to cover this

File: smoke-tests/spec/prometheusrules_spec.rb (hunk 2 above)
Comment (jasonBirchall) [reply smoke-tests/spec/prometheusrules_spec.rb:1]: new test added

**Review by poornima-krishnasamy — APPROVED** [review]

**Pull request merged**